org.opentosca.container.api/src/org/opentosca/container/api/controller/SituationsController.java

```diff
@@ -103,6 +106,20 @@ public Response createSituation(final SituationDTO situation) {
 
 		return Response.ok(instanceURI).build();
 	}
+	
+	@DELETE
+    @Consumes({ MediaType.TEXT_PLAIN })
+    @Path("/situations/{situation}")
```

Comment:
inconsistent indentation character (`@DELETE` has a tab, `@Consumes` and `@Path` use spaces, the method body is indented with a leading tab **and then** spaces...)